everestrs/everestrs/build.rs

```diff
@@ -24,8 +28,19 @@ fn find_everest_workspace_root() -> PathBuf {
 /// Returns the Libraries path if this is a standalone build of everest-framework or None if it is
 /// not.
 fn find_libs_in_everest_framework(root: &Path) -> Option<Libraries> {
-    let everestrs_sys = root.join("everest-framework/build/everestrs/libeverestrs_sys.a");
-    let framework = root.join("everest-framework/build/lib/libframework.so");
+    let (everestrs_sys, framework) =
+        if let Ok(everest_framework_dir) = env::var("EVEREST_RS_FRAMEWORK_BINARY_LOCATION") {
+            let everest_framework_path = PathBuf::from(everest_framework_dir);
+            (
+                everest_framework_path.join("everestrs/libeverestrs_sys.a"),
+                everest_framework_path.join("lib/libframework.so"),
+            )
+        } else {
+            (
+                root.join("everest-framework/build/everestrs/libeverestrs_sys.a"),
+                root.join("everest-framework/build/lib/libframework.so"),
+            )
+        };
```

Comment:
Not sure about rust best practices, but for me this `if let Ok(..) = ... {` doesn't naturally read.  Also the implicit returns are hard to spot (but could just be me ...)

Reply:
nit: I'd prefer a match statement here too. The implicit returns are rustic - i.e. idiomatic. They are like meaningful whitespace in Python - one first finds them appalling, once used to them they become rather nice.